go/ct/st/state.go

```diff
@@ -24,7 +24,8 @@ import (
 // Upper bound for gas, this limit is required since evmc defines a signed type for gas.
 // Limiting gas also solves issue 293 regarding out of memory failures,
 // discussed here: https://github.com/Fantom-foundation/Tosca/issues/293
-const MaxGas = tosca.Gas(1 << 60)
+// This should be enough gas to allow for ~450MB.
```

Comment:
It would be good to document how we got to this number for future readers of this code.